core/modules/widgets/navigator.js

```diff
@@ -112,22 +107,43 @@ NavigatorWidget.prototype.replaceFirstTitleInStory = function(storyList,oldTitle
 		storyList.splice(0,0,newTitle);
 	}
 };
-
+	
 NavigatorWidget.prototype.addToStory = function(title,fromTitle) {
 	var storyList = this.getStoryList();
-	if(storyList) {
-		// See if the tiddler is already there
-		var slot = this.findTitleInStory(storyList,title,-1);
-		// If not we need to add it
-		if(slot === -1) {
-			// First we try to find the position of the story element we navigated from
-			slot = this.findTitleInStory(storyList,fromTitle,-1) + 1;
-			// Add the tiddler
-			storyList.splice(slot,0,title);
-			// Save the story
-			this.saveStoryList(storyList);
+	// Quit if we cannot get hold of the story list
+	if(!storyList) return;
```

Comment:
TW coding style is to always uses braces with the "if" statement:

http://tiddlywiki.com/dev/#TiddlyWiki%20Coding%20Style%20Guidelines

Although reading it I see that it doesn't say so explicitly; I'll amend.